nautobot/dcim/tests/test_views.py

```diff
@@ -266,39 +275,50 @@ def setUpTestData(cls):
         cls.form_data = {
             "name": "Intermediate 2",
             "slug": "intermediate-2",
-            "parent": lt2.pk,
+            "parent": lt1.pk,
             "description": "Another intermediate type",
             "content_types": [ContentType.objects.get_for_model(Rack).pk, ContentType.objects.get_for_model(Device).pk],
         }
 
         cls.csv_data = (
             "name,slug,parent,description,content_types",
-            "Intermediate 3,intermediate-3,Root 1,Another intermediate type,ipam.prefix",
-            'Intermediate 4,intermediate-4,Root 1,Another intermediate type,"ipam.prefix,dcim.device"',
+            "Intermediate 3,intermediate-3,Root,Another intermediate type,ipam.prefix",
+            'Intermediate 4,intermediate-4,Root,Another intermediate type,"ipam.prefix,dcim.device"',
             "Root 3,root-3,,Another root type,",
         )
 
         cls.slug_source = "name"
-        cls.slug_test_object = "Intermediate 1"
+        cls.slug_test_object = "Root"
 
     def get_deletable_object_pks(self):
-        """To get the correct bulk-delete object count, make sure we avoid a cascade deletion."""
-        return [loctype.pk for loctype in list(LocationType.objects.filter(children__isnull=True))[:3]]
+        lts = [
+            LocationType.objects.create(name="Country"),
+            LocationType.objects.create(name="State"),
+            LocationType.objects.create(name="City"),
+        ]
+        return [lt.pk for lt in lts]
+
+    def get_deletable_object(self):
+        return LocationType.objects.create(name="LT delete")
+
+    def _get_queryset(self):
+        return super()._get_queryset().order_by("last_updated")
 
 
 class LocationTestCase(ViewTestCases.PrimaryObjectViewTestCase):
     model = Location
 
     @classmethod
     def setUpTestData(cls):
-        lt1 = LocationType.objects.create(name="Root Type 1")
-        lt2 = LocationType.objects.create(name="Intermediate Type 1", parent=lt1)
-        lt3 = LocationType.objects.create(name="Leaf Type 1", slug="leaf-1", parent=lt2, description="A leaf type")
+        lt1 = LocationType.objects.get(name="Building")
+        lt2 = LocationType.objects.get(name="Floor")
+        lt3 = LocationType.objects.get(name="Room")
+        lt3.description = "A leaf type"
         for lt in [lt1, lt2, lt3]:
             lt.validated_save()
 
         active = Status.objects.get(name="Active")
-        site = Site.objects.create(name="Site 1", slug="site-1", status=active)
+        site = Site.objects.first()
         tenant = Tenant.objects.create(name="Tenant 1")
```

Comment:
Not blocking, but this should be able to use the tenant fixture instead of creating a new one?